lighthouse-core/report/html/templates.html

```diff
@@ -783,24 +783,64 @@
 
 <template id="tmpl-lh-3p-filter">
   <style>
+    /* The input checkbox element is invisible but should span the same width and height as the label. */
     .lh-3p-filter {
-      background-color: var(--table-higlight-background-color);
       color: var(--color-gray-600);
       float: right;
       padding: 6px;
+      position: relative;
     }
-    .lh-3p-filter-label, .lh-3p-filter-input {
-      vertical-align: middle;
-      user-select: none;
+    .lh-3p-filter__input {
+      cursor: pointer;
+      opacity: 0;
+      width: 100%;
+      height: 100%;
+      z-index: 1;
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+    }
+    .lh-3p-filter__input:focus + label {
+      outline: -webkit-focus-ring-color auto 3px;
+    }
+
+    .lh-3p-filter__show, .lh-3p-filter__exclude {
+      min-width: 100px;
+      text-align: center;
+      background-color: var(--report-border-color-secondary);
+      padding: 0 8px 0 8px;
+      font-weight: 500;
+      font-size: 13px;
+      color: var(--color-gray-600);
+      /* Close the gap between the inline spans. */
+      float: left;
+    }
+    .lh-3p-filter__show {
+      border-radius: 20px 0 0 20px;
+    }
+    .lh-3p-filter__exclude {
+      border-radius: 0 18px 18px 0;
+    }
+    .lh-3p-filter__input:not(:checked) + label .lh-3p-filter__exclude,
+    .lh-3p-filter__input:checked + label .lh-3p-filter__show {
+      background-color: var(--color-blue-A700);
+      color: var(--color-white);
     }
-    .lh-3p-filter-input:disabled + .lh-3p-ui-string {
-      text-decoration: line-through;
+    .lh-3p-filter__input:disabled + label .lh-3p-filter__exclude {
+      color: var(--color-gray-400);
```

Comment:
👍

Reply:
was `cursor: not-allowed` shut down? I admit it is hard to tell that this is a disabled state when there's no immediate comparison nearby of what the enabled state looks like

Reply:
done